input/contribute/developer-experience/the-plan.md

```diff
@@ -6,8 +6,8 @@ ReactiveUI is the father of the extremely popular ReactiveCocoa AKA "RAC" (and a
 * https://github.com/ReactiveCocoa/ReactiveCocoa (17,675 stars, 3,289 forks)
 * https://github.com/ReactiveCocoa/ReactiveSwift
 
-IMHO (Geoff here) the .NET ecosystem is behind the eightball with reactive programming _even though we are the ecosystem that invented the reactive extensions_. This is super embarassing. Alot has changed in the nine years since the birth of the Reactive Extensions, there's been huge growth in the Javascript community (all angular developers, now must know reactive programming, the birth of the much loved ELM framework), Scala (server-side; play framework) and in partical the (Java "native") Android Mobile commuinity have turned to the reactive extnensions to help cope with the complexity of Android development. 
+IMHO (Geoff here) the .NET ecosystem is behind the eightball with reactive programming _even though we are the ecosystem that invented the reactive extensions_. This is super embarassing. Alot has changed in the nine years since the birth of the Reactive Extensions, there's been huge growth in the Javascript community (all angular developers, now must know reactive programming, the birth of the much loved ELM framework), Scala (server-side; play framework) and in particular the (Java "native") Android Mobile commuinity have turned to the reactive extensions to help cope with the complexity of Android development. 
```

Comment:
```suggestion
IMHO (Geoff here) the .NET ecosystem is behind the eight-ball with reactive programming _even though we are the ecosystem that invented the reactive extensions_. This is super embarrassing. A lot has changed in the nine years since the birth of the Reactive Extensions, there's been huge growth in the JavaScript community (all angular developers, now must know reactive programming, the birth of the much loved ELM framework), Scala (server-side; play framework) and in particular the (Java "native") Android Mobile community have turned to the reactive extensions to help cope with the complexity of Android development. 
```

Reply:
Found a couple more spelling mistakes in that paragraph.